Fix: Viewing skills

noisiver opened the pull request:
Added TYPE_SKILL as a define to be able to view skills.

The original version doesn't seem to use TYPE_SKILL so I have no reference to go by and therefore am not sure if this is the best way to do it or even what "category" to put it under. For now I added it above the profession skills.

Please do tell me where to put it if it's wrong and obviously I'll move it to the proper "category" in that file - or even if this is the wrong way to do it and I'll scrap it.

This does work on my end, for the record.

Helias commented:
Thanks a lot for your PR but this is wrong, the new version is using `Type::SKILL`, we probably need to replace some `TYPE_SKILL` into `Type::SKILL`.
So, please, replace `TYPE_SKILL` with `Type::SKILL`

noisiver commented:
> 

Thank you!

I used grep to find all files that has TYPE_SKILL in them and the only file I could find is the one I changed. It does work, as far as I can tell. I also removed TYPE_SKILL from defines before testing it.

Helias commented:
perfect! thanks a lot, I merged the PR
